pycloudlib/lxd/instance.py

```diff
@@ -109,28 +110,36 @@ def ip(self):
         while retries != 0:
             command = [
                 "lxc",
-                "list",
-                "^{}$".format(self.name),
-                "-c4",
-                "--format",
-                "csv",
+                "query",
+                f"/1.0/instances/{self.name}?recursion=1",
             ]
             result = subp(command)
             if result.ok and result.stdout:
-                ip_address = None
                 try:
-                    # Expect "<ip> (<interface>)" when network fully configured
-                    ip_address, _dev = result.stdout.split()
+                    info = json.loads(result.stdout)
                 except ValueError:
                     self._log.debug(
-                        "Unable to parse output of cmd: %s. Expected"
-                        " <ip> (<interface>), got: %s. Retrying %d time(s)...",
+                        "Unable to parse output of cmd: %s. Expected YAML,"
```

Comment:
Sorry, should have been JSON
```suggestion
                        "Unable to parse output of cmd: %s. Expected JSON,"
```